src/features/nameChecker/checkNameAgainstPatterns.ts

```diff
@@ -8,18 +10,20 @@ type Pattern = RuntimeConfigurationSchema['nameChecker']['patterns'][number]
 export function checkNameAgainstPatterns(
   name: string,
   patterns: Pattern[],
+  isZalgoEnabled: boolean,
   log: Logger = console,
-): RegExp | undefined {
+): boolean | undefined {
   for (const pattern of patterns) {
     try {
+      if (isZalgoEnabled) {
+        return isZalgo(name.trim())
+      }
```

Comment:
hm, if there are 5 patterns, the same code will run 5 times, and by enable zalgo check, it will disable pattern check.

did you mean:

```js
      if (isZalgoEnabled && isZalgo(name.trim())) {
        return true
      }
```

Reply:
Fix krub 🙇‍♂️